decision_service/examples/rl_sim_cpp/rl_sim.cc

```diff
@@ -0,0 +1,127 @@
+#include <iostream>
+#include <fstream>
+#include <boost/uuid/random_generator.hpp>
+#include <boost/uuid/uuid_io.hpp>
+
+#include "rl_sim.h"
+#include "ranking_response.h"
+#include "person.h"
+#include "live_model.h"
+#include "rl_sim_cpp.h"
+#include "config_utility.h"
+
+namespace r = reinforcement_learning;
+namespace u = reinforcement_learning::utility;
+namespace c = reinforcement_learning::utility::config;
+
+rl_sim::rl_sim(boost::program_options::variables_map vm):_options(std::move(vm)) {}
+std::string rl_sim::create_context_json(const std::string& cntxt, const std::string& action) {
+  std::ostringstream oss;
+  oss << "{ " << cntxt << ", " << action << " }";
+  return oss.str();
+}
+
+std::string rl_sim::create_uuid() {
+  return boost::uuids::to_string(boost::uuids::random_generator()( ));
+}
+
+int rl_sim::loop() {
+  if ( !init() ) return r::error_code::success;
```

Comment:
On line 77 there is another if statement without braces. What's the convention here since we have no newline here but there is one there.

Reply:
This particular instance is not relevant any more.  I have seen it used both ways in vw code base.  I add a new line if I think it makes the code more readable.